src/analysis/normed_space/star/multiplier.lean

```diff
@@ -0,0 +1,479 @@
+  star_add := λ x y, by {ext; simp only [star_fst, star_snd, add_fst, add_snd,
+    continuous_linear_map.add_apply, star_add]},
+  .. double_centralizer.has_star }
+
+instance : star_ring 𝓜(𝕜, A) :=
+{ star_mul := λ a b, by {ext; simp only [star_fst, star_snd, mul_fst, mul_snd, star_star,
+    continuous_linear_map.coe_mul, function.comp_app]},
+  .. double_centralizer.star_add_monoid }
+
+instance : star_module 𝕜 𝓜(𝕜, A) :=
+{ star_smul := λ k a, by {ext; exact star_smul _ _},
+  .. double_centralizer.star_add_monoid }
+
+end star
+
+/-!
+### Coercion from an algebra into its multiplier algebra
+-/
+
+noncomputable instance : has_coe_t A 𝓜(𝕜, A) :=
+{ coe := λ a,
+  { fst := continuous_linear_map.mul 𝕜 A a,
+    snd := (continuous_linear_map.mul 𝕜 A).flip a,
+    central := λ x y, mul_assoc _ _ _ } }
+
+@[simp, norm_cast]
+lemma coe_fst (a : A) : (a : 𝓜(𝕜, A)).fst = continuous_linear_map.mul 𝕜 A a := rfl
+@[simp, norm_cast]
+lemma coe_snd (a : A) : (a : 𝓜(𝕜, A)).snd = (continuous_linear_map.mul 𝕜 A).flip a := rfl
+
+/-- The coercion of an algebra into its multiplier algebra as a non-unital star algebra
+homomorphism. -/
+@[simps]
+def coe_hom [star_ring 𝕜] [star_ring A] [star_module 𝕜 A] [normed_star_group A] :
+  A →⋆ₙₐ[𝕜] 𝓜(𝕜, A) :=
+{ to_fun := λ a, a,
+  map_smul' := λ k a, by {ext; simp only [coe_fst, coe_snd, continuous_linear_map.map_smul,
+    smul_fst, smul_snd]},
+  map_zero' := by {ext; simp only [coe_fst, coe_snd, map_zero, zero_fst, zero_snd]},
+  map_add' := λ a b, by {ext; simp only [coe_fst, coe_snd, map_add, add_fst, add_snd]},
+  map_mul' := λ a b, by {ext; simp only [coe_fst, coe_snd, mul_apply', flip_apply, mul_fst, mul_snd,
+    continuous_linear_map.coe_mul, function.comp_app, mul_assoc]},
+  map_star' := λ a, by {ext; simp only [coe_fst, coe_snd, mul_apply', star_fst, star_snd,
+    flip_apply, star_mul, star_star]} }
+
+/-!
+### Norm structures
+We define the norm structure on `𝓜(𝕜, A)` as the pullback under
+`double_centralizer.add_group_hom_to_prod : 𝓜(𝕜, A) →+ (A →L[𝕜] A) × (A →L[𝕜] A)`, which provides
+a definitional isometric embedding. Consequently, completeness of `𝓜(𝕜, A)` is obtained by proving
+that the range of this map is closed.
+
+In addition, we prove that `𝓜(𝕜, A)` is a normed algebra, and, when `A` is a C⋆-algebra, we show
+that `𝓜(𝕜, A)` is also a C⋆-algebra. Moreover, in this case, for `a : 𝓜(𝕜, A)`,
+`‖a‖ = ‖a.fst‖ = ‖a.snd‖`. -/
+
+/-- The normed group structure is inherited as the pullback under the additive group monomoprhism
+`double_centralizer.to_prod : 𝓜(𝕜, A) →+ (A →L[𝕜] A) × (A →L[𝕜] A)` -/
+instance : normed_add_comm_group 𝓜(𝕜, A) :=
+normed_add_comm_group.induced _ _ (to_prod_hom : 𝓜(𝕜, A) →+ (A →L[𝕜] A) × (A →L[𝕜] A)) ext
+
+lemma norm_def (a : 𝓜(𝕜, A)) : ‖a‖ = max (‖a.fst‖) (‖a.snd‖) := rfl
+
+instance : normed_space 𝕜 𝓜(𝕜, A) :=
+{ norm_smul_le := λ k a, show max (‖k • a.fst‖) (‖k • a.snd‖) ≤‖k‖ * max (‖a.fst‖) (‖a.snd‖),
```

Comment:
```suggestion
{ norm_smul_le := λ k a, show max (‖k • a.fst‖) (‖k • a.snd‖) ≤ ‖k‖ * max (‖a.fst‖) (‖a.snd‖),
```

Reply:
Instead of using `show` here, can you rewrite by `norm_def`?

Reply:
sure, but's it's even easier just to provide the precise term, which I have done now.